### PR TITLE
Modernize universal tests

### PR DESCRIPTION
**Pull request by tylerwowen**
Remove `public`
Use `org.junit.jupiter.api.Assertions`

### CODE REVIEW

File: deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java [hunk 1]
```diff
@@ -9,11 +24,10 @@
 import static org.mockito.Mockito.when;
 
 import java.util.concurrent.TimeUnit;
-
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-public class GenericEventPublisherTest {
+class GenericEventPublisherTest {
```
Comment (ntascii) [inline deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java:30]: why remove? better to make it explicit?

**Comment by ntascii** [comment]
can you update the description with command used/ how you did to modernize the tests?

**Review by ntascii — APPROVED** [review]

File: deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java (hunk 1 above)
Comment (tylerwowen) [reply deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java:30]: It doesn't need to be public.

Comment (tylerwowen) [reply deploy-service/universal/src/test/java/com/pinterest/teletraan/universal/events/GenericEventPublisherTest.java:30]: https://rules.sonarsource.com/java/RSPEC-5810/

**Comment by tylerwowen** [comment]
Change base to master, as this no longer depend on the stack.

**Review by Cjpilbdev — APPROVED** [review]

**Pull request merged**